COMETwebapp/COMETwebapp/Pages/Login.razor.css

```diff
@@ -0,0 +1,14 @@
+﻿.btn-success {
```

Comment:
name of file suggests that this css is for the login page, shouldn't you have one central css?

Reply:
Maybe it is easier to control css for each component this way, but I can put all in a central css